spring-cloud-build/pom.xml

```diff
@@ -125,6 +128,13 @@ limitations under the License.
 					<artifactId>asciidoctor-maven-plugin</artifactId>
 					<version>1.5.2</version>
 				</plugin>
+				<plugin>
+					<groupId>org.apache.maven.plugins</groupId>
+					<artifactId>maven-surefire-plugin</artifactId>
+					<configuration>
+						<redirectTestOutputToFile>true</redirectTestOutputToFile>
```

Comment:
I'll test something tomorrow (8pm here) to make sure it is easily
overrideable. In the meantime, if you can point me to all CI confs that
need to be looked at

On Thu, Aug 13, 2015 at 7:30 PM, Spencer Gibb <notifications@github.com>
wrote:

> In spring-cloud-build/pom.xml
> <https://github.com/spring-cloud/spring-cloud-build/pull/16#discussion_r37001352>
> :
>
> > @@ -125,6 +128,13 @@ limitations under the License.
> >  					<artifactId>asciidoctor-maven-plugin</artifactId>
> >  					<version>1.5.2</version>
> >  				</plugin>
> > +				<plugin>
> > +					<groupId>org.apache.maven.plugins</groupId>
> > +					<artifactId>maven-surefire-plugin</artifactId>
> > +					<configuration>
> > +						<redirectTestOutputToFile>true</redirectTestOutputToFile>
>
> We removed -Dmaven.test.redirectTestOutputToFile=true where it was
> problematic. I just want to be able to see test output. I don't care how.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/spring-cloud/spring-cloud-build/pull/16/files#r37001352>
> .
>